### DISCUSSION TITLE
Send Budibase DB information to Firestore

### DISCUSSION THREAD

**Discussion by robertwatson1** [Budibase/budibase#7500] [Help] [answered]
I am building an application in Flutter. I've managed to build out a database using the Budibase DB, and I can see that I can retrieve this data using a HTTPS GET request. I was just wondering if I cans end the Budibase DB data through to Firestore? and then pull the information from Firestore into Flutter?

I only ask due to the strong integrations Flutter has with Firestore and Firebase.

**Comment by melohagan** [comment, marked as answer]
Hi @robertwatson1 

There are docs for the Firestore connector here: https://docs.budibase.com/docs/firestore
You should be able to perform the usual CRUD operations. 

It's worth noting however that is is specifically a Firestore connector, using this library: https://cloud.google.com/nodejs/docs/reference/firestore/latest/

We're NOT using the broader Firebase SDK: https://firebase.google.com/docs/reference/node/
